contracts/TokenConverter/AbstractTokenConverter.sol

```diff
@@ -1126,16 +1127,30 @@ abstract contract AbstractTokenConverter is AccessControlledV8, IAbstractTokenCo
         uint256 tokenOutUnderlyingPrice = priceOracle.getPrice(tokenAddressOut);
 
         uint256 incentive = configuration.incentive;
-        if ((address(converterNetwork) != address(0)) && (converterNetwork.isTokenConverter(msg.sender))) {
+
+        bool isPrivateConversion = address(converterNetwork) != address(0) &&
+            converterNetwork.isTokenConverter(msg.sender);
+        if (isPrivateConversion) {
             incentive = 0;
         }
 
         /// conversion rate after considering incentive(conversionWithIncentive)
         uint256 conversionWithIncentive = MANTISSA_ONE + incentive;
-        tokenInToOutConversion = (tokenInUnderlyingPrice * conversionWithIncentive) / tokenOutUnderlyingPrice;
 
         /// amount of tokenAddressIn after considering incentive(i.e. amountInMantissa will be less than actual amountInMantissa if incentive > 0)
-        amountInMantissa = ((amountOutMantissa * EXP_SCALE) + tokenInToOutConversion - 1) / tokenInToOutConversion; //round-up
+        if (isPrivateConversion) {
+            amountInMantissa =
+                (amountOutMantissa * tokenOutUnderlyingPrice * EXP_SCALE) /
+                (tokenInUnderlyingPrice * conversionWithIncentive);
+        } else {
+            amountInMantissa =
+                ((amountOutMantissa * tokenOutUnderlyingPrice * EXP_SCALE) +
+                    (tokenInUnderlyingPrice * conversionWithIncentive) -
+                    1) /
+                (tokenInUnderlyingPrice * conversionWithIncentive); //round-up
```

Comment:
Wdyt if we extract

```
function _divRoundingUp(uint256 numerator, uint256 denominator) internal {
  return (numerator + denominator - 1) / denominator;
}
```
?

This way we can make the intent more clear:
```suggestion
            amountInMantissa = _divRoundingUp(
                amountOutMantissa * tokenOutUnderlyingPrice * EXP_SCALE,
                tokenInUnderlyingPrice * conversionWithIncentive
            );
```

Reply:
Done https://github.com/VenusProtocol/protocol-reserve/pull/71/commits/a5560325bfe89c505aea5ca57d1ed6d1b6799ef1